### DISCUSSION TITLE
Styling

### DISCUSSION THREAD

**Discussion by TheITCloudGuy** [okieselbach/Autopilot-Manager#10] [Q&A] [answered]
Hi All,

First off want to say I love the project. It's fantastic. I was wondering if it is possible to do custom styling? I want to incorporate it with my current tooling and wasn't quite sure if the SLN files where made availbale other than the Artifiacts. Is this possible at all with the files available to the public?

Regardless, Great project so thank you for that.

**Comment by okieselbach** [comment, marked as answer]
First of all, thanks for the nice words! For the time being I wanted to leave the code in my hand and I'm taking care of it. I'm open for requests and suggestions. Lately I added also delete capabilities (request Intune device deletion, only with approval mode enabled) because of this change for re-enrollment of pre-provisioned device scenarios: https://docs.microsoft.com/en-us/mem/autopilot/troubleshoot-device-enrollment. 
Nevertheless, if someone needs access for security review purpose or something else, I'm more than willing to support these asks. If this helps to get clearance to use it for example. But maybe I'll change that strategy in future. I need to find my way also with all this.
